test/e2e-sno-disruptive/sno_disruptive_test.go

```diff
@@ -75,7 +76,10 @@ func waitForClusterInGoodState(t testing.TB, cs clientSet, waitPollTimeout, must
 			}
 		}
 
-		if time.Since(startTs) > mustBeReadyFor {
+		ckasoAvailable := v1helpers.IsOperatorConditionTrue(ckaso.Status.Conditions, "StaticPodsAvailable")
+		ckasoProgressing := v1helpers.IsOperatorConditionFalse(ckaso.Status.Conditions, "NodeInstallerProgressing")
```

Comment:
i would rename it to `ckasoNotProgressing`

Reply:
Updated.